tdrs-backend/tdpservice/parsers/schema_defs/tanf/t5.py

```diff
@@ -15,66 +16,129 @@
           preparsing_validators=[
               validators.hasLength(71),
           ],
-          postparsing_validators=[],
+          postparsing_validators=[
+              validators.if_then_validator(
+                  condition_field='FAMILY_AFFILIATION', condition_function=validators.matches(1),
+                  result_field='SSN', result_function=validators.validateSSN()
+                  ),
+              validators.validate__FAM_AFF__SSN(),
+              validators.if_then_validator(
+                  condition_field='FAMILY_AFFILIATION', condition_function=validators.isInLimits(1, 3),
+                  result_field='RACE_HISPANIC', result_function=validators.validateRace()
```

Comment:
@elipe17 i think this may still need review. 0 is not a valid response for this condition. 

```
def validateRace():
    """Validate race."""
    return make_validator(
        lambda value: value >= 0 and value <= 2,
        lambda value: f'{value} is not greater than or equal to 0 or smaller than or equal to 1.'
```